src/responseHandlers/AbstractResponseHandler.js

```diff
@@ -33,6 +36,27 @@ var AbstractResponseHandler = jsface.Class([EventEmitter], {
 	},
 
 	_printResponse: function(error, response, body, request) {
+        if (Globals.outputFileVerbose) {
+            var filepath = path.resolve(Globals.outputFileVerbose);
+
+            var requestString = "-------------------------------------------------------------------------------------------\n" +
```

Comment:
Might be better to use a template for this, say [Mustache](https://github.com/janl/mustache.js/).